packages/cli/package.json

```diff
@@ -1,10 +1,17 @@
 {
   "name": "@bucketco/cli",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "packageManager": "yarn@4.1.1",
   "description": "CLI for Bucket service",
   "main": "./dist/index.js",
   "type": "module",
+  "license": "MIT",
+  "author": "Bucket Co.",
```

Comment:
```suggestion
  "author": "Bucket.",
```

Reply:
With the dot?

Reply:
Kept the dot

Reply:
hah, sorry, we should remove the dot. Apologies!